pipelines/_ingredients/ApPipe.yaml

```diff
@@ -74,6 +75,10 @@ subsets:
       - rbClassify
       - transformDiaSrcCat
       - diaPipe
+  qaMetrics:
+    subset:
+      - analyzeAssocDiaSrcCore
+      - diffimTaskCore
```

Comment:
Why not define `qaMetrics` in `apDetectorVisitQualityCore.yaml`? Does `analysis_tools` have rules/conventions against it?

Is this subset supposed to exclude `analyzeTrailedDiaSrcCore`?

Reply:
Please include all tasks in `apPipe` so that the alias behaves correctly in higher-level pipelines like `ApVerify`. Unfortunately, Middleware doesn't want subsets to be included as subset members, so you'll have to do it manually.